src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2164,6 +2164,9 @@ because xbuild doesn't support framework reference assemblies.
       Condition=" '%(_BuildTargetAbis.Identity)' == 'armeabi' "
       Text="Invalid value 'armeabi' in %24(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties to remove the old value. If the properties page does not show an 'armeabi' checkbox, un-check and re-check one of the other ABIs and save the changes."
   />
+  <AndroidError Code="XA1025"
+      ResourceName="XA1025"
+      Condition=" '$(AndroidAotModeValidateAbi)' != 'false' AND '%(_BuildTargetAbis.Identity)' == 'armeabi-v7a' AND '$(AndroidAotMode)' == 'Hybrid' " />
```

Comment:
Setting `$(AndroidAotModeValidateAbi)`=`false` will now disable the error. I think a "public" property is OK because the property is expected to be API-stable until the error is no longer relevant (that is, until `$(AndroidAotMode)`=`Hybrid` produces apps that run successfully on armeabi-v7a). Because `AndroidAotMode` is not documented, I also left the new property likewise undocumented. I'm happy to adjust either of these decisions.

I debated a little bit on the name of the property. In particular, I wondered whether it would be preferable to word the property in the "negative" like `$(AndroidAotModeSkipValidateAbi)` because the default behavior right now is that validation is enabled when `AndroidAotModeValidateAbi` is `null`, which might be a tiny bit counter-intuitive. Keeping the property in the "positive" keeps the name shorter though, so overall I decided I'd keep it that way for now.

Reply:
I think if we make `AndroidAotModeValidateAbi` is a public property we need to document it. We should probably document `AndroidAotMode` as well but that could probably be done in another PR maybe..